src/main/java/edu/harvard/iq/dataverse/AbstractGlobalIdServiceBean.java

```diff
@@ -77,7 +77,20 @@ public String getIdentifierForLookup(String protocol, String authority, String i
         metadata.put("datacite.publisher", producerString);
         metadata.put("datacite.publicationyear", generateYear(dvObjectIn));
         return metadata;
-    }  
+    } 
+    
+    protected HashMap<String, String> addMetadataForDestroyedDataset(DvObject dvObjectIn) {
+        HashMap<String, String> metadata = new HashMap<>();
+        String authorString = ":unav";
+        String producerString = ":unav";
+        String titleString = "This item has been removed from publication";
+
+        metadata.put("datacite.creator", authorString);
+        metadata.put("datacite.title", titleString);
+        metadata.put("datacite.publisher", producerString);
+        metadata.put("datacite.publicationyear", "9999");
```

Comment:
I agree with @bencomp, that special strings should generally be defined as constants somewhere. 
Also, I find this a little confusing that this method is in AbstractGlobalIdServiceBean - but it does something that's very specific to the Datacite metadata. I understand you are modeling this method after the already existing method above (addBasicMetadata()) that does the same thing... I'm assuming we do this in the "abstract" bean because this datacite metadata format is used by both the Datacite and EZID DOI implementations. I still feel this could be confusing for the next person working with this code; could you rename these methods, to somehow indicate that they are specific to DOI implementations - but not to Handles (that extends the same Abstract service class); something like addDOIMetadataForDestroyedDataset() etc.?